support/iconsClassesGenerator.js

```diff
@@ -0,0 +1,116 @@
+const fs = require('fs');
+const path = require('path');
+const sharp = require('sharp');
+
+const configFilePath = process.argv[2];
+const outputFile = process.argv[3];
+
+const rawData = fs.readFileSync(configFilePath);
+const iconsConfig = JSON.parse(rawData);
+
+const sourceIconsPath = path.join(
+	path.dirname(configFilePath),
+	'..',
+	iconsConfig.dir
+);
+
+const activeColor = '#272833';
+const defaultColor = '#6b6c7e';
+const disableColor = '#a7a9bc';
+const focusColor = '#272833';
+const hoverColor = '#272833';
+
+const encodeSvgData = (svgData, color) => {
+	const symbols = /[\r\n%#()<>?\[\\\]^`{|}]/g;
+
+	return svgData
+		.replace(/(<svg[^>]*)( fill="[^"]+")/, '\1')
+		.replace(/<svg/, `<svg fill="${color}"`)
+		.replace(/"/g, "'") // Use single quotes instead of double to avoid encoding.
+		.replace(/>\s{1,}</g, '><')
+		.replace(/\s{2,}/g, ' ')
+		.replace(symbols, encodeURIComponent)
+		.replace(/\s+/g, '%20');
+};
+
+const getCSS = (svgData, cKEditorIcon, direction) => {
+	let directionClass = direction ? `.cke_${direction}` : '';
+
+	const activeIconCSS = `${directionClass}.cke_hidpi .cke_button.cke_button_on .cke_button__${cKEditorIcon}_icon,
+		${directionClass} .cke_button.cke_button_on .cke_button__${cKEditorIcon}_icon {
+			background: url("data:image/svg+xml;charset=utf8,${encodeSvgData(
+				svgData,
+				activeColor
+			)}") !important;
+		}`;
```

Comment:
nit: I'd format these so that the *output* is consistently indented:

```
	const activeIconCSS = `
		${directionClass}.cke_hidpi .cke_button.cke_button_on .cke_button__${cKEditorIcon}_icon,
		${directionClass} .cke_button.cke_button_on .cke_button__${cKEditorIcon}_icon {
			background: url("data:image/svg+xml;charset=utf8,${encodeSvgData(
				svgData,
				activeColor
			)}") !important;
		}
	`;
```

(The way you have it, you have 0 indent on first line, then it jumps to two tabs of indent)